### PR TITLE
Prepare 4.2.2 Release

### PR DESCRIPTION
**Pull request by jww3**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v4.2.2
+* `url-helper.ts` now leverages well-known environment variables. by @jww3 in https://github.com/actions/checkout/pull/1941
```
Comment (joshmgross) [inline CHANGELOG.md:4]:
```suggestion
* `url-helper.ts` now leverages well-known environment variables by @jww3 in https://github.com/actions/checkout/pull/1941
```

**Review by joshmgross — APPROVED** [review]

**Review by joshmgross — APPROVED** [review]

**Comment by AdrianFlores688** [comment]
Sarah

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v4.2.2
+* `url-helper.ts` now leverages well-known environment variables by @jww3 in https://github.com/actions/checkout/pull/1941
```
Comment (marissareyes80) [inline CHANGELOG.md:4]: https://www.facebook.com/profile.php?id=100078106500760

**Review by marissareyes80 — APPROVED** [review]
https://www.facebook.com/profile.php?id=100078106500760

**Review by marissareyes80 — COMMENTED** [review]
Screenshot_20241105_233702_Lite
Screenshot_20241105_233652_Lite
https://www.facebook.com/profile.php?id=100078106500760

File: CHANGELOG.md (hunk 1 above)
Comment (Nanlonely) [reply CHANGELOG.md]: Help

**Review by Domi83s — APPROVED** [review]

**Review by blg666 — APPROVED** [review]

**Pull request merged**